schema/fda_drugs_enum_schema.mcf

```diff
@@ -0,0 +1,2236 @@
+Node: dcid:AdministrationRouteEnum
```

Comment:
Wow this is a lot of enums. Are all enum instances in the file manually curated or auto-generated?  If the latter, it might be worth adding a comment and moving the auto-generated one to a separate file (away from enum classes).

Reply:
yes, they are all auto generated. I can move the enum classes into fda_drugs_schema.mcf and keep the auto generated enums in the fda_drugs_enum_schema.mcf file. Does that work?